File: src/main/Java/boj_step10/boj27323.java
import java.io.*;

public class boj27323 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int height = Integer.parseInt(reader.readLine());
        int width = Integer.parseInt(reader.readLine());
        writer.append(String.valueOf(height * width)).close();
    }
}

File: src/main/boj_step10/boj27323.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val a = r.readLine().toInt()
    val b = r.readLine().toInt()
    w.write((a * b).toString())
    w.close()
}